atomicfu/build.gradle

```
@@ -2,6 +2,7 @@
  * Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

```suggestion
import org.jetbrains.kotlin.gradle.tasks.KotlinJvmCompile
```

Better to use types from KGP-api